srml/staking/src/lib.rs

```diff
@@ -267,16 +313,27 @@ decl_storage! {
 	}
 	add_extra_genesis {
 		config(stakers): Vec<(T::AccountId, T::AccountId, BalanceOf<T>)>;
+		config(nominators): Vec<(T::AccountId, Vec<T::AccountId>)>;
```

Comment:
this should be folded into `stakers`: add an extra enum field `enum { Validator, Nominator(Vec<AccountId>) }` to discern which is which.